src/rate/coinbase.js

```diff
@@ -45,8 +48,14 @@ export function makeCoinbasePlugin (
           toCurrency,
           rate
         })
+        if (DUPLICATE_RATE_MAP[toCurrency]) {
```

Comment:
This was supposed to be a *new* rate provider. It shouldn't affect any of the existing plugins in any way. The new plugin's `fetchRates` method would just return some fixed JSON data.